common/static/common/js/discussion/views/discussion_thread_view.js

```diff
@@ -91,7 +91,6 @@
                     id = self.model.get('id');
                     if (collection.get(id)) {
                         self.model = collection.get(id);
```

Comment:
I'm not sure what this is for, but it looks like it's invoked [here](https://github.com/edx/edx-platform/blob/bjacobel/TNL-5249/common/static/common/js/discussion/discussion.js#L140) and [here](https://github.com/edx/edx-platform/blob/bjacobel/TNL-5249/common/static/common/js/discussion/discussion_module_view.js#L167).